internal/cli/cmd/connectivity.go

```diff
@@ -120,6 +121,9 @@ func newCmdConnectivityTest() *cobra.Command {
 	cmd.Flags().BoolVarP(&params.Debug, "debug", "d", false, "Show debug messages")
 	cmd.Flags().BoolVarP(&params.PauseOnFail, "pause-on-fail", "p", false, "Pause execution on test failure")
 	cmd.Flags().BoolVar(&params.SkipIPCacheCheck, "skip-ip-cache-check", true, "Skip IPCache check")
+	cmd.Flags().BoolVar(&params.Perf, "perf", false, "Run network Performance tests")
+	cmd.Flags().IntVar(&params.Duration, "perf-duration", 30, "Duration for the Performance test to run, Default:30s")
+	cmd.Flags().IntVar(&params.Samples, "perf-samples", 1, "Number of Performance samples to capture (how many times to run each test), Default:1")
```

Comment:
Let's drop the default value from the usage here as well:

```suggestion
	cmd.Flags().IntVar(&params.Samples, "perf-samples", 1, "Number of Performance samples to capture (how many times to run each test)")
```